prac_09/unreliable_car.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+from prac_09.car import Car
+
+
+class UnreliableCar(Car):
+    """Specialised version of a Car that includes reliability."""
+
+    def __init__(self, name, fuel, reliability):
+        """Initialise a UnreliableCar instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        """Drive a given distance depending on the reliability."""
+        if random.uniform(0, 100) < self.reliability:
```

Comment:
You correctly used the random.uniform(0, 100) method to generate a random number between 0 and 100 for determining if the car should drive - Nice work! 😃